src/main/java/teammates/storage/api/FeedbackQuestionsDb.java

```diff
@@ -150,13 +150,28 @@ public FeedbackQuestionAttributes getFeedbackQuestion(
     
     /**
      * Updates the feedback question identified by `{@code newAttributes.getId()} 
+     * without keeping the old timestamp
      * For the remaining parameters, the existing value is preserved 
      *   if the parameter is null (due to 'keep existing' policy).<br> 
+     * 
      * Preconditions: <br>
      * * {@code newAttributes.getId()} is non-null and
      *  correspond to an existing feedback question. <br>
      */
     public void updateFeedbackQuestion(FeedbackQuestionAttributes newAttributes) throws InvalidParametersException, EntityDoesNotExistException {
+        updateFeedbackQuestion(newAttributes, false);
+    }
+    
+    /**
+     * Updates the feedback question identified by `{@code newAttributes.getId()} 
+     * For the remaining parameters, the existing value is preserved 
+     *   if the parameter is null (due to 'keep existing' policy).<br> 
+     * The timestamp is preserved or not based on the value of {@code keepUpdateTimestamp} 
```

Comment:
Maybe "The timestamp for its last update is not dependent on newAttributes and is preserved depending on keepUpdateTimestamp" (add formatting)?
Or any other line that explicitly mentions that it is the updatedAt timestamp and that it is not affected by the `newAttributes` passed in.

For the other comment for the previous `updateFeedbackQuestion` (L151+), it may be good to just explicitly say that the update timestamp will be set to the time of update or something to the tune of that.

Does how the methods treat the creation timestamp have to be mentioned in either of them?
I believe it doesn't change even if `newAttributes` has a different creation time?
If so, it may help stating that for clarity.